sci-libs/dtk/dtk-2017.02.16.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils
+
+DESCRIPTION="dtk is a meta-platform for modular scientific platform development"
+HOMEPAGE="https://github.com/d-tk/dtk"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+SRC_URI="https://timeraider4u.github.io/distfiles/files/${P}.tar.gz"
```

Comment:
Well, the git tags/releases are sometimes quite behind and I already experienced issues when these projects were not "in sync" (namely compilation errors). And changes in the latest git may result in patching errors, therefore I prefer running a cron-job for building source tar-balls to have a known point in time from which I can develop the ebuilds. But I am up to suggestions on how to improve this situation.